ActionsRouter.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express')
+const actionModel = require('./data/helpers/actionModel')
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+    try {
+        const actions = await actionModel.get();
+        res.status(200).json(actions);
```

Comment:
Missing a check in case of the actions array comes back empty, it's always a good practice to get used to checking for all edge cases if we want to build robust APIs. This is just a recommendation.